lib/streamlit/caching/__init__.py

```diff
@@ -14,8 +14,25 @@
 import contextlib
 from typing import Iterator
 
-from .memo_decorator import MEMO_CALL_STACK, _memo_caches, MemoAPI
-from .singleton_decorator import SINGLETON_CALL_STACK, _singleton_caches, SingletonAPI
+from google.protobuf.message import Message
```

Comment:
I noticed that DeltaGenerator only imports this during typechecking time, and probably I should do the same with all these new instances of importing it